storage-node/packages/discovery/publish.js

```diff
@@ -1,6 +1,5 @@
-const ipfsClient = require('ipfs-http-client')
-
-const ipfs = ipfsClient('localhost', '5001', { protocol: 'http' })
+// const ipfsClient = require('ipfs-http-client')
```

Comment:
Please, remove or comment the commented code.

Reply:
done in d607a24f8eb7591013a5aae44a506fe63268bafa